web/js/map/ui.js

```diff
@@ -971,7 +956,10 @@ export default function mapui(models, config, store, ui) {
         $zoomOut.button('enable');
       }
     };
-    map.getView().on('change:resolution', lodashDebounce(debouncedZoomChange, 30));
+    map.getView().on('change:resolution', () => {
```

Comment:
I think the denounce leading option https://lodash.com/docs/#debounce might have been another way to solve. Just FYI for future reference.  `shouldComponentUpdate` seems like a good solution.